pkg/json.go

```diff
@@ -49,11 +49,12 @@ type jsonCheckDocumentationV2 struct {
 
 //nolint:govet
 type jsonCheckResultV2 struct {
-	Details []string                 `json:"details"`
-	Score   int                      `json:"score"`
-	Reason  string                   `json:"reason"`
-	Name    string                   `json:"name"`
-	Doc     jsonCheckDocumentationV2 `json:"documentation"`
+	Details               []string                 `json:"details"`
+	Score                 int                      `json:"score"`
+	Reason                string                   `json:"reason"`
+	Name                  string                   `json:"name"`
+	Doc                   jsonCheckDocumentationV2 `json:"documentation"`
+	MaintainersAnnotation []string                 `json:"maintainersAnnotation"`
```

Comment:
as written, this would break the bigquery table. Need to think about how to not do that. Maybe some form of `omitempty`?

maybe `jsonCheckResultV3`?

Reply:
It would add the `maintainersAnnotation` to the JSON output.
https://go.dev/play/p/kwtvfXV61cM

I was thinking if combined with the experimental flag, this would prevent the struct from showing up in any output before we're ready for the feature.

> This will also not impact the cronjob or bigquery table as the JSON results are defined elsewhere.

Agreed. I forgot the cron handles JSON output separately.

Reply:
> I was thinking if combined with the experimental flag, this would prevent the struct from showing up in any output before we're ready for the feature.

Oh, I think I got it now. Thanks for the playground example! I will check it out and add the `omitempty` field.

Reply:
Done. Will review when testing.